### ISSUE TITLE
Static IP address assignment in OpenVPN mode

### ISSUE THREAD

**Issue by zhilchenko** [SoftEtherVPN/SoftEtherVPN#138]
I need assign static IP to OpenVPN client. 
Problem: openvpn client won't connect (disconnect after PUSH_REQUEST) to server when SecureNAT with Virtual DHCP is disabled. If I enable it, client get random IP which cannot be overridden in client config file.

I can set up DHCP and assign IP by MAC, but this solution ruins whole idea of all-in-one server. Is there anything to solve problem? 

Ability to get rid of requirement of VirtualDHCP would be ideal. Or maybe there is openvpn config option to override server pushed IP?

**Comment by moatazelmasry2** [comment]
Yes of course you can push from your OpenVPN clients whatever routes. 

See this thread: https://serverfault.com/questions/631037/how-to-route-only-specific-openvpn-traffic-through-a-openvpn-based-on-ip-filteri

Specifically:

```
# redirect all default traffic via the VPN
push redirect-gateway def1
# redirect the Intranet network 192.168.1/24 via the VPN
route 192.168.1.0 255.255.255.0
# redirect another network to NOT go via the VPN
route 10.10.0.0 255.255.255.0 net_gateway
# redirect a host using a domainname to NOT go via the VPN
route www.google.ca 255.255.255.255 net_gateway
```

Does this help?
